test/e2e/end_user_test.go

```diff
@@ -51,4 +85,65 @@ var _ = Describe("Openshift on Azure end user e2e tests [EndUser]", func() {
 		// Expect(kerrors.IsForbidden(err)).To(Equal(true))
 		fmt.Printf("PDB create error: %v\n", err)
 	})
+
+	It("should deploy a template and check the visit counter increments", func() {
+		var regex = regexp.MustCompile(`\<span.*id=\"count-value\"\>(?P<counter>\d+)\<\/span\>`)
```

Comment:
check your backslashes, I don't think you need them before `<`, `>`, `"` or `/`.
also `span.*id` is a bit dodgy - just start at `id`
also you're not using `(?P<counter>\d+)` - just do `(\d+)`